wcomponents-lde/src/main/java/com/github/bordertech/wcomponents/lde/PlainLauncher.java

```diff
@@ -124,19 +116,7 @@ public InterceptorComponent createInterceptorChain(final Object request) {
 		// The toolkit must render itself within the page shell
 		// and wrap the main WComponent output.
 		InterceptorComponent.replaceInterceptor(PageShellInterceptor.class,

```

Comment:
As this is now replacing itself with itself, this override method is no longer needed.